po/lxd.pot

```diff
@@ -1413,7 +1413,7 @@ msgstr  ""
 msgid   "taken at %s"
 msgstr  ""
 
-#: lxc/exec.go:163
+#: lxc/exec.go:185
```

Comment:
Commit doesn't seem needed, there's no string change and that change in line number shouldn't trip Jenkins.